TELCODOCS-1388: Add IPI troubleshooting information to our documentation (#73672)
openshift/openshift-docs: modules/ipi-install-troubleshooting-troubleshooting-ssh-access-to-cluster-nodes.adoc

// This module is included in the following assemblies: 
//
// installing/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc

:_content-type: PROCEDURE
[id="troubleshooting-ssh-access-to-cluster-nodes_{context}"]
= Troubleshooting SSH access to cluster nodes

For added security, you cannot SSH into the cluster from outside the cluster by default. However, you can access control plane and worker nodes from the provisioner node. If you cannot SSH into the cluster nodes from the provisioner node, the nodes might be waiting on the bootstrap VM. The control plane nodes retrieve their boot configuration from the bootstrap VM, and they will not boot successfully if they do not retrieve the boot configuration.

.Procedure

> . If you have physical access to the nodes, check their console output to determine if they have successfully booted or are still retrieving their boot configuration. Control plane nodes waiting a long time for their boot configuration is indicative of problems on the bootstrap VM.

Reviewer: ```suggestion
. If you have physical access to the nodes, check their console output to determine if they have successfully booted. If the nodes are still retrieving their boot configuration, there might be problems with the bootstrap VM .
```
Does this change the meaning too much? I got a little lost in the repetition of the original sentence. Feel free to disregard if it isn't helpful.
